supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/GcpBatchBackendSingletonActor.scala

```diff
@@ -1,38 +1,82 @@
 package cromwell.backend.google.pipelines.batch
 
 import akka.actor.{Actor, ActorLogging, Props}
-import cromwell.core.WorkflowId
-import scala.concurrent.ExecutionContext
-import cromwell.backend.google.pipelines.batch.GcpBatchRequestFactory.CreatePipelineParameters
+import com.google.cloud.batch.v1.{BatchServiceClient, GetJobRequest, JobName}
+import cromwell.core.Dispatcher.BackendDispatcher
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
 
 object GcpBatchBackendSingletonActor {
-  def props(name: String) = Props(new GcpBatchBackendSingletonActor(name))
+  def props(name: String) = Props(new GcpBatchBackendSingletonActor(name)).withDispatcher(BackendDispatcher)
 
-  case class GcpBatchRequest(workflowId: WorkflowId, createParameters: CreatePipelineParameters,jobName: String, gcpBatchCommand: String, gcpBatchParameters: CreateGcpBatchParameters)
-  case class BatchGetJob(jobId: String, projectId: String, region: String)
-  case class BatchJobAsk(test: String)
+  // This is the only type of messages that can be processed by this actor from this actor
+  sealed trait Action extends Product with Serializable
```

Comment:
I have refactored the actor messages to be clear on what can be handled and what can be produced, it is a pattern to keep the actors organized because messages are untyped